data/autoyast_sle15/autoyast_bcache.xml

```diff
@@ -0,0 +1,218 @@
+        <arch>{{ARCH}}</arch>
+      </addon>
+      <addon>
+        <name>sle-module-desktop-applications</name>
+        <version>{{VERSION}}</version>
+        <arch>{{ARCH}}</arch>
+      </addon>
+    </addons>
+  </suse_register>
+  <bootloader>
+      <global>
+          <timeout config:type="integer">-1</timeout>
+      </global>
+  </bootloader>
+  <general>
+    <mode>
+      <confirm config:type="boolean">false</confirm>
+    </mode>
+  </general>
+  <keyboard>
+    <keyboard_values>
+      <delay/>
+      <discaps config:type="boolean">false</discaps>
+      <numlock>bios</numlock>
+      <rate/>
+    </keyboard_values>
+    <keymap>english-us</keymap>
+  </keyboard>
+  <language>
+    <language>en_US</language>
+    <languages/>
+  </language>
+  <ntp-client>
+    <ntp_policy>auto</ntp_policy>
+  </ntp-client>
+  <partitioning config:type="list">
+    <drive>
+      <device>/dev/vda</device>
+      <type config:type="symbol">CT_DISK</type>
+      <use>all</use>
+      <enable_snapshots config:type="boolean">true</enable_snapshots>
+      <partitions config:type="list">
+        <partition>
+          <filesystem config:type="symbol">btrfs</filesystem>
+          <mount>/</mount>
+          <create config:type="boolean">true</create>
+          <size>70%</size>
+        </partition>
+        <partition>
+          <mount>/swap</mount>
+          <create config:type="boolean">true</create>
+          <filesystem config:type="symbol">swap</filesystem>
+          <size>auto</size>
+        </partition>
+        <partition>
+          <bcache_backing_for>/dev/bcache0</bcache_backing_for>
+          <create config:type="boolean">true</create>
+          <format config:type="boolean">false</format>
+          <size>max</size>
+        </partition>
+      </partitions>
+    </drive>
+    <drive>
+      <type config:type="symbol">CT_DISK</type>
+      <device>/dev/vdb</device>
+      <use>all</use>
+      <disklabel>none</disklabel>
+      <partitions config:type="list">
+        <partition>
+          <bcache_caching_for config:type="list">
+            <listentry>/dev/bcache0</listentry>
+          </bcache_caching_for>
+          <create config:type="boolean">false</create>
+          <format config:type="boolean">false</format>
+          <resize config:type="boolean">false</resize>
+        </partition>
+      </partitions>
+    </drive>
+    <drive>
+      <type config:type="symbol">CT_BCACHE</type>
+      <device>/dev/bcache0</device>
+      <bcache_options>
+        <cache_mode>writethrough</cache_mode>
+      </bcache_options>
+      <use>all</use>
+      <partitions config:type="list">
+        <partition>
+          <mount>/home</mount>
+          <size>max</size>
+        </partition>
+        <partition>
+          <mount>swap</mount>
+          <filesystem config:type="symbol">swap</filesystem>
+          <size>auto</size>
+        </partition>
+      </partitions>
+    </drive>
+  </partitioning>
+  <software>
```

Comment:
this section 'software' you need to figure out what you can remove, perhaps you need some pattern, but many things will be default too. Check other minimal profiles, 'mini' 'multi-btrfs', recent ones.

Reply:
i think thats quite ok as it is.

Reply:
it is not, we don't store cloned profile there, we store minimal AutoYaST profiles. I'm not sure if I can give you all the reason for that, but from the top of my head I believe the most important is that if something change in how AutoYaST generate those defaults some tags not related with the goal of the profile itself will not remains the same, so as less data we have less test maintenance, the other one is just that is more useful for us looking at other profiles to continue creating minimal ones and they are less polluted.
It is a little tricky and you have to go through the process several time to know what to pull out, but then you understand better the defaults.

Reply:
@jknphy i think you gave me a very good explanation but the software section is the same with other AY profiles. definetely i need to study those cases but i cant see serious reason to change them now. Mainly for the reason above which IMO means i would have to take care the others too.

Reply:
Check [mini profile](https://github.com/os-autoinst/os-autoinst-distri-opensuse/blob/master/data/autoyast_sle15/mini.xml) for minimal software section, some packages and pattern are installed by default.

Reply:
I get it. What i am objecting to, is that the xml is minimal enough as it is. The packages that they are in the xml will be installed anyway. Anyway i reduced them.

Reply:
looks much more better now, thanks!